tests/playwright-test/timeout.spec.ts

```diff
@@ -68,6 +68,35 @@ test('should respect test.setTimeout', async ({ runInlineTest }) => {
   expect(result.output).toContain('Timeout of 1000ms exceeded');
 });
 
+test('should respect test.setTimeout outside of the test', async ({ runInlineTest }) => {
+  const result = await runInlineTest({
+    'a.spec.ts': `
+      const { test } = pwt;
+
+      test.setTimeout(500);
+      test('fails', async ({}) => {
+        await new Promise(f => setTimeout(f, 1000));
+      });
+      test('passes', async ({}) => {
+        await new Promise(f => setTimeout(f, 100));
+      });
+
+      test.describe('suite', () => {
+        test.setTimeout(50);
+        test('fails', async ({}) => {
+          await new Promise(f => setTimeout(f, 100));
+        });
+        test('passes', async ({}) => {
+        });
+      });
```

Comment:
Whats the intended/actual behavior of this code
```ts
test.setTimeout(50);
test.describe(() => {
        test('unknown', async ({}) => {
          await new Promise(f => setTimeout(f, 100));
        });
});
test.setTimeout(500);
```

Reply:
Actual behavior is 500 for all tests. What would you expect?